src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
Should I understand that you suggest?

"More than one matching entity. Please refine your query"

Ok for me. 

For me, more than the particular wording is to make sure the same text is used in all places where it makes sense and the best way of ensuring that is using a #define. That way, if we change our mind and decide to change, we only need to change in one place (i.e. the #define value).